pkg/clusterconditions/promql/promql.go

```diff
@@ -89,11 +104,14 @@ func (p *PromQL) Valid(ctx context.Context, condition *configv1.ClusterCondition
 func (p *PromQL) Match(ctx context.Context, condition *configv1.ClusterCondition) (bool, error) {
 	// Lookup the address every attempt in case the service IP changes.  This can happen when the thanos service is
 	// deleted and recreated.
-	address, err := p.Address(ctx)
+	host, err := p.Host(ctx)
 	if err != nil {
 		return false, fmt.Errorf("failure determine thanos IP: %w", err)
 	}
-	clientConfig := api.Config{Address: address}
+	p.url.Host = host
+
+	klog.V(2).Infof("evaluating cluster conditions at: %s", p.url.String())
```

Comment:
Consider bumping the level here, seems to me we'd be too verbose at normal. This does not seem particularly interesting event?

Reply:
I have decided to drop this log entirely. The log stays the same (or different IP addresses are seen) for the duration of the run. For this I could just add logs showing the passed flags for the CVO but this is not needed. And for the networking logs (that would be interesting), these logs would be produced by the prometheus client and we do not control these logs (just the potential returned errors).